lib/fluent/plugin/in_tail.rb

```diff
@@ -892,16 +967,35 @@ def opened?
         def handle_notify
           with_io do |io|
             begin
+              number_bytes_read = 0
+              start_reading = Fluent::EventTime.now
               read_more = false
 
               if !io.nil? && @lines.empty?
                 begin
                   while true
                     @fifo << io.readpartial(8192, @iobuf)
                     @fifo.read_lines(@lines)
-                    if @lines.size >= @read_lines_limit
+
+                    number_bytes_read += @lines.last ? @lines.last.size : 0
```

Comment:
I think it should be `number_bytes_read += 8192`. see comment https://github.com/fluent/fluentd/pull/3185#issuecomment-754545966

Reply:
Thanks! I’ll rewrite to use 8192 here.

Reply:
Fixed in https://github.com/fluent/fluentd/pull/3185/commits/7a542c7405eae248a0d01189969efa238b5f4bd7.